### PR TITLE
Update python malware scripts for additional graph queries

### PR DESCRIPTION
**Pull request by tpott**
Added a couple new scripts in the malware/ directory that can fetch malware samples based on indicator IDs or family IDs.

### CODE REVIEW

File: malware/family_variants.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    creds = get_app_creds()
+    args = get_parser().parse_args()
+    # inputs
+    if args.input is None and args.inputlist is None:
+        print('Input list or input file is needed')
+        sys.exit(-1)
+    elif args.input is not None and args.inputlist is not None:
+        print('Can only take hex hash or input file')
+        sys.exit(-1)
+    elif args.input is not None:
```
Comment (jessek) [inline malware/family_variants.py]: There's no need to use elif here. The alternative is exiting the program, so no worries.

File: malware/family_variants.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            # TODO verify line.strip() is a uint64
+            query_ids.append(line.strip())
+    elif args.inputlist is not None:
+        query_ids = set(map(int, args.inputlist.split(',')))
+    # output directories
+    if args.output is not None:
+        outdir = args.output
+    else:
+        # TODO stdout
+        outdir = '.'
```
Comment (jessek) [inline malware/family_variants.py]: I don't understand what needs to be done here.

File: malware/family_variants.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    if not os.path.isdir(outdir):
+        print('%s is not a directory' % outdir)
+        sys.exit(-1)
+    # time based query
+    if args.since is not None:
+        print('--since not yet implemented')
+        sys.exit(-1)
+    if args.until is not None:
+        print('--until not yet implemented')
+        sys.exit(-1)
```
Comment (jessek) [inline malware/family_variants.py]: If they're not implemented yet, don't include them as options.

File: malware/related_samples.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      print('Total time elapsed: %d seconds' % (end - start))
+
+if __name__ == '__main__':
+    creds = get_app_creds()
+    args = get_parser().parse_args()
+    # inputs
+    if args.input is None and args.inputlist is None:
+        print('Input list or input file is needed')
+        sys.exit(-1)
+    elif args.input is not None and args.inputlist is not None:
```
Comment (jessek) [inline malware/related_samples.py]: Again, no need for elif.

**Comment by jessek** [comment]
Thanks for contributing to ThreatExchange! I have a few comments above, but in general your code looks good.

**Comment by jessek** [comment]
I'd be happy to merge if you can take out those elifs!

**Comment by tpott** [comment]
GitHub is hard >_<

**Comment by jessek** [comment]
Great stuff, thanks!

**Pull request merged**